### ISSUE TITLE
.eln archive import

### ISSUE THREAD

**Issue by nmontua** [wagenadl/eln#10]
Hi, not sure how easy this is to implement, but it would be valuable to be able to import .eln archives (open MTI-license zip format https://github.com/TheELNConsortium/TheELNFileFormat/blob/master/SPECIFICATION.md) from other ELNs. I would like to use notedeln to work on entries, when no Internet is available or for personal backups from our cloud based ELN. Thank your for your time!

**Comment by wagenadl** [comment]
Interoperability is generally a good thing, but the file format you 
mention looks to me rather abstract. Are you aware of any actual 
software that uses it? It seems to me that the specification would have 
to be extended quite a bit before it would cover the formatting 
requirements of an actual notebook.

In particular, can you give me more details about your cloud based ELN?

**Comment by nmontua** [comment]
Fair point. In my case it is Elabftw which is very popular in academia as far as ELNs go (in europe anyway). Unsurprisingly, the principal developer of elabftw Nicolas CARPi (https://github.com/NicolasCARPi) is also involved with the .eln Github. I thought about cobbling something together in python to display the .eln archives, but your software is pretty impressive as it is, so I thought I could suggest implementing it.

**Comment by wagenadl** [comment]
Would you be able to send me / share with me some entries from your 
Elabftw notebook to exemplify the range of content that you'd to import 
into NotedELN?

**Comment by nmontua** [comment]
Absolutely, I can prepare some dummy entries  with some pictures etc. that would reflect a typical entry. If you want to make some for yourself: there is a hosted live demo, that anyone can access where you can make entries and export them if you care to take a look (https://demo.elabftw.net/login.php). However, I am of course willing to send to said entries. Thank you for considering my feature request and happy holidays.
